src/styles/_exchange-form.scss

```
@@ -191,3 +191,20 @@
   color: white;
   font-size: 14px;
 }
+
+.account-select {
+  .v-select__selection {
+    font-size: 15px;
+  }
+}
+
+.amount-input {
+  font-size: 15px;
+}
+
+.wallet-address-input {
+  font-size: 11px;
+  .v-input__slot {
+    padding: 0 5px !important;
+  }
+}
```

Recall leaving an empty line at the end of your files
```suggestion
}

```